model/src/order.rs

```diff
@@ -345,6 +352,8 @@ pub struct OrderMetaData {
     pub executed_fee_amount: BigUint,
     pub invalidated: bool,
     pub status: OrderStatus,
+    #[serde(with = "h160_hexadecimal")]
+    pub contract_version: H160,
```

Comment:
naming nit: I'd call this field `settlement_contract` everywhere to be clear about what contract we are talking.

Reply:
Do you mean `settlement_version` or should I go back and also change #906 to also use `settlement_contract` as well? I think I will do that. Use settlement_contract everywhere.